### PR TITLE
pre-filter fedmsg events before hitting the queue

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: packit_service/worker/jobs.py [hunk 1]
```diff
@@ -49,6 +49,16 @@ def add_to_mapping(kls: Type["JobHandler"]):
     return kls
 
 
+def do_we_process_fedmsg_topic(topic: str) -> bool:
+    """ do we process selected fedmsg topic? """
+    global JOB_NAME_HANDLER_MAPPING
```
Comment (jpopelka) [inline packit_service/worker/jobs.py]: [nitpicking]: do we need this when we don't change the value? (we just iterate over it)

File: packit_service/worker/jobs.py [hunk 2]
```diff
@@ -49,6 +49,16 @@ def add_to_mapping(kls: Type["JobHandler"]):
...
+    for kls in JOB_NAME_HANDLER_MAPPING.values():
+        t = getattr(kls, "topic", "this-is-not-a-real-topic")
+        if t == topic:
```
Comment (jpopelka) [inline packit_service/worker/jobs.py]: Just wondering if it's more readable for you with the intermediary `t`
(than just `if getattr(kls, "topic", "not-a-real-topic") == topic:`

File: packit_service/worker/jobs.py (hunk 1 above)
Comment (TomasTomecek) [reply packit_service/worker/jobs.py]: I don't know.

File: packit_service/worker/jobs.py (hunk 2 above)
Comment (TomasTomecek) [reply packit_service/worker/jobs.py]: but then we need to catch the exc; ok, let me change

**Comment by TomasTomecek** [comment]
:P

**Pull request merged**